deepmd/entrypoints/freeze.py

```diff
@@ -45,6 +45,8 @@ def _make_node_names(model_type: str, modifier_type: Optional[str] = None) -> Li
         "model_attr/tmap",
         "model_attr/model_type",
         "model_attr/model_version",
+        "min_nbor_dist",
+        "training_script",
```

Comment:
I suggest the following changes:
`"min_nbor_dist"` -> `"train/min_nbor_dist"`
`"training_script"` -> `"train/training_script"`